src/components/footer/footer.html

```diff
@@ -0,0 +1,9 @@
+<footer class="footer section">
+  <div class="container">
+    <span class="footer__copyright">&copy; 2020 | All Rights Reserved |</span>
+    <span class="footer__copyright">Developed with<img class="footer__image" src="./components/footer/img/footer-heart.svg" alt=""> by
+      <a class="footer__link" href="#" data-modal-team>GoIT Students</a></span>
```

Comment:
Co do znaczników w tym kodzie nie wiem czy nie zrobiłabym <p> zamiast <span> a zamiast <a> <span>.  Co na to Dawid?